src/05-derivatives/05-functional-derivatives.md

```diff
@@ -0,0 +1,390 @@
+# Functional derivatives
+
+## Functional derivatives maps
+
+A derivative map is a measure derived from a functional series, mapped onto spatial
+locations as defined by voxels in a volume or vertices on a surface.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<label>/
+        func/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.{nii[.gz],gii,dscalar.nii}
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_desc-<label>]_<suffix>.json
+```
+
+for example:
+
+```Text
+pipeline1/
+    sub-001/
+        func/
+            sub-001_task-rest_run-1_space-MNI305_mean.nii.gz
+            sub-001_task-rest_run-1_space-MNI305_mean.json
+```
+
+The following table lists allowed suffixes and their corresponding measures:
+
+| `<suffix>`       | Measure                                                          |
+| ---------------- | ---------------------------------------------------------------- |
+| `mean`           | Mean across the temporal/4th dimension of the data               |
+| `std`            | Standard deviation across the temporal/4th dimension of the data |
+| `tsnr`           | Temporal SNR (i.e. mean / std)                                   |
+| `sfs`            | Signal fluctuation sensitivity                                   |
+| `alff`           | Amplitude low frequency fluctuations                             |
+| `falff`          | Fractional amplitude of low frequency fluctuations               |
+| `reho`           | Regional homogeneity (voxelwise only)                            |
+| `dcb`, `dcw`     | Voxelwise degree centrality binary and weighted                  |
+| `ecb`, `ecw`     | Voxelwise eigenvector centrality binary and weighted             |
+| `lfcdb`, `lfcdw` | Local functional connectivity density                            |
+| `vmhc`           | Voxel mirrored homotopic connectivity                            |
+
+The following metadata JSON fields are valid for derivative maps:
+
+| Key name       | Description                                                               | Required for suffix        |
+| -------------- | ------------------------------------------------------------------------- | -------------------------- |
+| BandpassFilter | String describing all relevant parameters of the applied bandpass filter. | `alff`, `falff`            |
+| Neighborhood   | String describing neighborhood for regional measures.                     | `reho`                     |
+| Threshold      | String describing threshold used for determining graph edges.             | `dcb`, `dcw`, `ecb`, `ecw` |
+| Method         | String describing method used to calculate measure.                       | `dcb`, `dcw`, `ecb`, `ecw` |
```

Comment:
`BandpassFilter` should be conformed to electrophysiological derivatives. cc @sappelhoff for guidance.

Not sure if there are equivalents for these others.

Reply:
we currently specify filtering properties in an object of objects: https://bids-specification.readthedocs.io/en/latest/glossary.html#softwarefilters-metadata

example:

```JSON
{
    "SoftwareFilters":{
        "Anti-aliasing filter":{
            "half-amplitude cutoff (Hz)":500,
            "Roll-off":"6dB/Octave"
        },
        "Some other filter":{
            "some other parameter":"some value"
        }
    }
}
```

Reply:
https://bids-specification--519.org.readthedocs.build/en/519/derivatives/functional-derivatives.html#sidecar-json-_boldmapjson_cbvmapjson

![image](https://github.com/bids-standard/bids-specification/assets/83442/03621b34-e5f3-4091-8811-fb5008e42e0c)